bot/cogs/tags.py

```diff
@@ -209,10 +216,13 @@ def _command_on_cooldown(tag_name) -> bool:
             if tag_name:
                 log.debug(f"{ctx.author} requested the tag '{tag_name}'")
                 embed.title = tag_name
-                self.tag_cooldowns[tag_name] = {
-                    "time": time.time(),
-                    "channel": ctx.channel.id
-                }
+
+                # Cooldown
```

Comment:
imho, I don't think this comment provides a lot of value. I can see that this is *something to do with cooldown*, but I can also see that by reading the if statement body. maybe something like `# Set the cooldown for this tag on this channel` would be more intuitive, although I don't know whether this really needs a comment anyways.